src/libs/ReportUtils.js

```diff
@@ -969,6 +970,7 @@ function buildOptimisticWorkspaceChats(policyID, policyName) {
     );
     const announceChatReportID = announceChatData.reportID;
     const announceReportActionData = buildOptimisticCreatedReportAction(announceChatData.ownerEmail);
+    const announceCreatedReportActionID = announceReportActionData.reportActionID;
```

Comment:
It does not look to me like this will work. If you look at the object returned by `buildOptimisticCreatedReportAction`, it (confusingly) has a shape like:

```js
function buildOptimisticCreatedReportAction() {
    return {
        0: {
            reportActionID: '1234',
            ...
            ...
        },
    };
}
```

So if you do this:

```js
const createdReportActionData = buildOptimisticCreatedReportAction();

// This will forsure be undefined
const createdReportActionID_wrong = createdReportActionData.reportActionID;

// This is what you need
const createdReportActionID = createdReportActionData[0].reportActionID;
```

FWIW this is something I am changing as part of https://github.com/Expensify/App/pull/13118.